src/Miso.hs

```diff
@@ -48,8 +48,6 @@ module Miso
     -- * Html
   , module Miso.Html
   , module Miso.Render
-    -- * Mathml
-  , module Miso.Mathml
```

Comment:
I think it doesn't make much sense to export Mathml modules from the top-level Miso module.
1. it will be likely relatively rarely used
2. there's bunch of attributes that clashes with html/svg attributes - so if users do want to math stuff, they should import it separately.